components/mpas-ocean/src/Registry.xml

```diff
@@ -2411,6 +2568,9 @@
 			 description="time tendency of normal component of velocity"
 			 packages="forwardMode"
 		/>
+                <var name="tendVerticalVelocity" type="real" dimensions="nVertLevelsP1 nCells Time" units="m s^{-2}" name_in_code="verticalVelocityTend"
+                         description="time tendency of the vertical velocity (Only active for non hydrostatic mode)"
+                />
```

Comment:
Add to` nonhydrostaticPKG`?

I think having the name be `tendVerticalVelocity` but `name_in_code` be `verticalVelocityTend` is too confusing.  For consistency with other variables in this pool, the `name_in_code` should be `verticalVelocity`.